### ISSUE TITLE
OutOfBoundsDatetime Exception when returning Pandas df via API

### ISSUE THREAD

**Issue by obsidianforensics** [google/timesketch#869]
This occurs when using the API to return a Pandas df, when the results have a pre 1970-01-01 timestamp (1601-01-01T00:00:00+00:00 is probably most common problem time). 

Example query on demo server:
```
active_sketch = sketches[0]
active_sketch.explore(query_string='bepbmhgboaologfdajaanbcjmnhjmhfn', as_pandas=True)

> ...
> OutOfBoundsDatetime: Out of bounds nanosecond timestamp: 1601-01-01 00:00:00
```

I think that pandas.to_datetime() can't handle dates that early (or webkit dates at all?).

**Comment by berggren** [comment]
@kiddinn I'm assigning this to you to take a look.

**Comment by kiddinn** [comment]
this got fixed by #992 

Basically adding a try/except, so there is an attempt to make the timestamp using to_datetime and if it does not succeed we fall back to just the string timestamp.
